ts/features/it-wallet/screens/presentation/ItwPrQrCodeScreen.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import { Image, SafeAreaView, View } from "react-native";
+import RNQRGenerator from "rn-qr-generator";
+import { IOStyles, LabelSmall, VSpacer } from "@pagopa/io-app-design-system";
+import { useNavigation } from "@react-navigation/native";
+import ItwLoadingSpinner from "../../components/ItwLoadingSpinner";
+import { useOnFirstRender } from "../../../../utils/hooks/useOnFirstRender";
+import { IOStackNavigationProp } from "../../../../navigation/params/AppParamsList";
+import { ItwParamsList } from "../../navigation/ItwParamsList";
+import I18n from "../../../../i18n";
+import BaseScreenComponent from "../../../../components/screens/BaseScreenComponent";
+import ItwKoView from "../../components/ItwKoView";
+import { getItwGenericMappedError } from "../../utils/errors/itwErrorsMapping";
+
+// A mocked QR code to be used in the proximity flow
+// TODO: remove this mocked QR code after the proximity flow is implemented [SIW-688]
+const mockedQrCode =
+  "mdoc:owBjMS4wAYIB2BhYS6QBAiABIVggUCnUgO0nCmTWOkqZLpQJh1uO2Q0YCTbYtUowBJU6ltEiWCBPkYpJZpEY4emfmR_2eFS5XQN68wihmgEoiMVEf8M3_gKBgwIBowD0AfULUJr9sL_rAkZCk114baNK4rY";
```

Comment:
Should we move this in our `mocks.ts` file in order to keep them all in one place?